src/main/java/htsjdk/tribble/index/IndexFactory.java

```diff
@@ -224,7 +229,20 @@ else if (indexFile.endsWith(FileExtensions.TABIX_INDEX)) {
      * @param codec     the codec to use for decoding records
      */
     public static LinearIndex createLinearIndex(final File inputFile, final FeatureCodec codec) {
-        return createLinearIndex(inputFile, codec, LinearIndexCreator.DEFAULT_BIN_WIDTH);
+        return createLinearIndex(
```

Comment:
You can use IOUtil.toPath(file) which does a null safe conversion to path and then just have the null check in the final one.

Reply:
Oh yeah - done.